packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -1,19 +1,19 @@
 {
-  "name": "@spotify-backstage/plugin-{{id}}",
-  "version": "{{version}}",
-  "main": "dist/cjs/index.js",
-  "types": "dist/cjs/index.d.ts",
-  "license": "Apache-2.0",
-  "private": false,
-  "scripts": {
-    "build": "backstage-cli plugin:build",
-    "lint": "backstage-cli plugin:lint",
-    "test": "backstage-cli plugin:test"
-  },
-  "devDependencies": {
-    "@spotify-backstage/cli": "^{{version}}"
-  },
-  "dependencies": {
-    "@material-ui/lab": "4.0.0-alpha.45"
-  }
+"name": "@spotify-backstage/plugin-{{id}}",
+"version": "{{version}}",
+"main": "dist/cjs/index.js",
+"types": "dist/cjs/index.d.ts",
+"license": "Apache-2.0",
+"private": false,
+"scripts": {
+"build": "backstage-cli plugin:build",
+"lint": "backstage-cli lint",
+"test": "backstage-cli test"
+},
+"devDependencies": {
+"@spotify-backstage/cli": "^{{version}}"
+},
+"dependencies": {
+"@material-ui/lab": "4.0.0-alpha.45"
+}
```

Comment:
Some prettier sadness here. As it is a hbs file..

Reply:
Oh derp, thanks :grin:

Reply:
Fixed it